amacc.c

```diff
@@ -1720,11 +1756,12 @@ int elf32(int poolsz, int *main)
     append_strtab(&data, "");
     char *libc = append_strtab(&data, "libc.so.6");
     char *ldso = append_strtab(&data, "libdl.so.2");
+    char *ldccso = append_strtab(&data, "libgcc_s.so.1");
```

Comment:
How about renaming it to `libgcc`?

Reply:
GCC low level library named by libgcc_s.
Should we change it?

Reply:
`libgcc_s` sounds good to me.